app/data/mysql/migrations.go

```diff
@@ -69,3 +72,18 @@ func createAccountLastLoginAtField(db *sqlx.DB) error {
 	}
 	return err
 }
+
+func createAccountTOTPFields(db *sqlx.DB) error {
+	_, err := db.Exec(`
+        ALTER TABLE accounts ADD totp_secret VARCHAR(255) DEFAULT NULL
+    `)
+	if mysqlError, ok := err.(*mysql.MySQLError); ok {
+		if mysqlError.Number == 1060 { // 1060 = Duplicate column name
+			err = nil
+		}
+	}
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
>i think it'd be fine to add the columns in a batch DDL and simplify the migrations somewhat. i'm not sure how someone could end up in a partially migrated state after once this is finished and merged, unless they ctrl+c at the magical moment during migrations.

Could I get some clarification for this? Should I combine all the migrations into one batch?